### PR TITLE
Add French translation

### LINKED ISSUE #52: How to internationalize the grid?
## Question  
I was reading https://www.infragistics.com/products/ignite-ui-angular/angular/components/localization.html and I'd like to confirm, before taking this route, this is the way to do it.
Should I really create a variable with all resources of the grid, manually translated, per language? Aren't there no default resources for "common" languages like German, French, Italian, etc? Only Japanese and Korean by downloading the i18n package?

Is this correct or am I missing something?

 * igniteui-angular version: 8.1.0
 * browser: Firefox

### PR DESCRIPTION
**Pull request by DanielRose**
Closes #52 

With the help of some colleagues I created a translation into French. Since I'm not a native speaker myself, please excuse any errors.

### CODE REVIEW

**Comment by kdinev** [comment]
For reviewers reference, here's the EN strings:

https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/core/i18n

**Comment by DanielRose** [comment]
Rebased on current master

**Comment by DanielRose** [comment]
Rebased on current master

File: src/i18n/FR/grid-resources.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+const GridResourceStringsFR_: ExpandRequire<IGridResourceStrings> = {
+  igx_grid_groupByArea_message: 'Faites glisser un en-tête de colonne et déposez-le ici pour le regrouper par cette colonne.',
+  igx_grid_emptyFilteredGrid_message: 'Aucun enregistrement trouvé.',
+  igx_grid_emptyGrid_message: 'La grille n\'a pas de données.',
+  igx_grid_filter: 'Filtre',
+  igx_grid_filter_row_close: 'Fermer',
+  igx_grid_filter_row_reset: 'Éteindre',
+  igx_grid_filter_row_placeholder: 'Ajouter une valeur de filtre',
+  igx_grid_filter_row_boolean_placeholder: 'Tout',
+  igx_grid_filter_row_date_placeholder: 'Choisisser la date',
```
Comment (Lipata) [inline src/i18n/FR/grid-resources.ts]:
```suggestion
  igx_grid_filter_row_date_placeholder: 'Choisissez la date',
```

File: src/i18n/FR/grid-resources.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  igx_grid_excel_search_placeholder: 'Chercher',
+  igx_grid_excel_select_all: 'Tout sélectionner',
+  igx_grid_excel_select_all_search_results: 'Sélectionner tous les résultats de la recherche',
+  igx_grid_excel_add_to_filter: 'Ajouter la sélection actuelle au filtre',
+  igx_grid_excel_blanks: '(Blancs)',
+  igx_grid_excel_hide: 'Masquer la colonne',
+  igx_grid_excel_show: 'Afficher la colonne',
+  igx_grid_excel_pin: 'Fixer la colonne',
+  igx_grid_excel_unpin: 'Détacher la colonne',
+  igx_grid_excel_select: 'Sélectionnez la colonne',
```
Comment (Lipata) [inline src/i18n/FR/grid-resources.ts]:
```suggestion
  igx_grid_excel_select: 'Sélectionner la colonne',
```

File: src/i18n/FR/grid-resources.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  igx_grid_advanced_filter_ungroup: 'Supprimer le regroupement',
+  igx_grid_advanced_filter_delete: 'Supprimer',
+  igx_grid_advanced_filter_delete_filters: 'Supprimer les filtres',
+  igx_grid_advanced_filter_initial_text: 'Commencer par créer un groupe de conditions liées à "Et" ou "Ou"',
+  igx_grid_advanced_filter_column_placeholder: 'Sélectionner la colonne',
+  igx_grid_advanced_filter_value_placeholder: 'Valeur',
+  igx_grid_pinned_row_indicator: 'Attachée',
+  igx_grid_hiding_check_all_label: 'Cacher tout',
+  igx_grid_hiding_uncheck_all_label: 'Afficher tout',
+  igx_grid_pinning_check_all_label: 'Attacher tout',
```
Comment (Lipata) [inline src/i18n/FR/grid-resources.ts]:
```suggestion
  igx_grid_pinning_check_all_label: 'Attache tout',
```

**Review by Lipata — CHANGES_REQUESTED** [review]
@DanielRose thank you once again for the French translations! Because we don't have a French-speaking colleague, I've just checked the changes in the PR with google translate and it only suggested me 3 things that I've listed below. I guess those suggestions can be wrong or just an alternative, so please approve them if they are only typos or are a better fit.

File: src/i18n/FR/grid-resources.ts (hunk 1 above)
Comment (DanielRose) [reply src/i18n/FR/grid-resources.ts]: I simplified to "Choisir la date"

File: src/i18n/FR/grid-resources.ts (hunk 2 above)
Comment (DanielRose) [reply src/i18n/FR/grid-resources.ts]: Fixed

File: src/i18n/FR/grid-resources.ts (hunk 3 above)
Comment (DanielRose) [reply src/i18n/FR/grid-resources.ts]: Fixed

**Comment by DanielRose** [comment]
Rebased on current master

**Review by IvayloG — APPROVED** [review]

**Review by Lipata — APPROVED** [review]

**Pull request merged**